thirty-three-across/thirty-three-across-htb-system-tests.js

```diff
@@ -0,0 +1,166 @@
+'use strict';
+
+function getPartnerId() {
+    return 'ThirtyThreeAcrossHtb';
+}
+
+function getStatsId() {
+    return 'THI';
+}
+
+function getBidRequestRegex() {
+    return {
+        method: 'POST',
+        urlRegex: /ssc\.33across\.com\/api\/v1\/hb/
+    };
+}
+
+function getCallbackType() {
+    return 'NONE';
+}
+
+function getArchitecture() {
+    return 'MRA';
+}
+
+function getConfig() {
+    return {
+        siteId: 'acbdefgABCDEFG-1234567',
+        test: 1,
+        xSlots: {
+            1: {
+                sizes: [[300, 250], [160, 600]],
+                productId: 'siab',
+                bidfloor: 0.1
+            }
+        }
+    };
+}
+
+function validateBidRequest(request) {
+    var config = getConfig();
+    var xSlot = config.xSlots[1];
+    var sizes = xSlot.sizes;
+
+    var queryObj = JSON.parse(request.body);
+
+    expect(queryObj.id).toBeDefined();
+
+    expect(queryObj.imp.length).toEqual(1);
+    expect(queryObj.imp[0].id).toBeDefined();
+    expect(queryObj.imp[0].banner.format[0]).toEqual({
+        w: sizes[0][0],
+        h: sizes[0][1],
+        ext: { }
+    });
+    expect(queryObj.imp[0].banner.format[1]).toEqual({
+        w: sizes[1][0],
+        h: sizes[1][1],
+        ext: { }
+    });
+    expect(queryObj.imp[0].bidfloor).toEqual(xSlot.bidfloor);
+    expect(queryObj.imp[0].ext.ttx.prod).toEqual(xSlot.productId);
+
+    expect(queryObj.site).toBeDefined();
+    expect(queryObj.site.id).toEqual(config.siteId);
+    expect(queryObj.site.page).toBeDefined();
+    expect(queryObj.site.ref).toBeDefined();
+
+    expect(queryObj.user).toBeDefined();
+    expect(queryObj.user.ext).toBeDefined();
+
+    expect(queryObj.regs).toBeDefined();
+    expect(queryObj.regs.ext).toBeDefined();
+
+    expect(queryObj.ext).toBeDefined();
+    expect(queryObj.ext.ttx).toBeDefined();
+    expect(queryObj.ext.ttx.prebidStartedAt).toBeDefined();
```

Comment:
with changing to indexStartedAt, we need to update ttx, i prefer just keep using the `prebidStartedAt` as the generic name, because by change to some other generic name, we also need to change prebidjs adapter

Reply:
Okay, I suppose this fine especially given the caller info which will specifies which client it is.